src/utils/get-config.test.ts

```diff
@@ -22,7 +22,7 @@ describe('#getConfig(true)', () => {
 		const config1 = await getConfig(true);
 		const config2 = await getConfig();
 
-		expect(config1.yarnMutexFilePath).toStrictEqual(config2.yarnMutexFilePath);
+		expect(config1).toBe(config2);
```

Comment:
This change causes this test to no longer prove that it skips reconstructing the config.

Reply:
`toBe` checks for referential equality, when we reconstruct the config we create a new object, so the two objects would not have referential equality if it was rebuilt each time.

Reply:
Ah, okay, I thought it did a shallow comparison, rather than checking for referential equality.

Reply:
Behind the scenes it's using `Object.is` according to [this piece of documentation](https://jestjs.io/docs/expect#tobevalue).

One thing we could do differently if we wanted to, which they talk about in their documentation at the same link, is use something like `expect(Object.is(config2, config1))toEqual(true)` since in that test all we care about is referential equality, and if the `toBe` test fails it will report a deep comparison which won't be very useful. That sort of splitting hairs though.

Reply:
Hmm, that may be wise, to make it clear that all we care about is referential integrity. It's important that when tests fail, their output clearly indicates what went wrong, and I can imagine the current implementation would be pretty misleading.